packages/teams-js/src/public/webStorage.ts

```diff
@@ -25,12 +27,34 @@ export namespace webStorage {
       throw errorNotSupportedOnPlatform;
     }
 
+    if (
+      (GlobalVars.hostClientType === HostClientType.android || GlobalVars.hostClientType === HostClientType.ios) &&
```

Comment:
Should we also return true for HostClientType.ipados?

Reply:
Fixed; good catch.